js/src/components/SnippetPreviewSection.js

```diff
@@ -17,6 +17,22 @@ const Section = styled( StyledSection )`
 	}
 `;
 
+/**
+ * Function to map the editor data to data for the preview.
```

Comment:
Can you extend the documentation to explain what's happening a bit more? Without much context about what's happening in yoast-components, it's quite confusing that the input already is 'mapped' (according to the argument name).

Reply:
Was looking a bit into this and wondering why we're trying to format this string in the plugin. In `yoast-components` there's already a function `mapDataToPreview` which returns the `mappedData ` object used here. As I see it, the yoast-component should be responsible for passing proper data that the plugin should just present. Also, if we fix the formatting in the plugin, the standalone example on yoast-component won't work and it will still show `%20`.

Reply:
@afercia I asked @atimmer this. And in `yoast-components` the workings should be generic. The dashes whitespaces to dashes is WordPress specific. That is why I implemented it in the plugin itself.